### ISSUE TITLE
no vmware provider

### ISSUE THREAD

**Issue by cybacolt** [saltstack-formulas/salt-formula#186]
as of 2015.8.x, vmware as a provider is available, as such i've manually altered the formula to add that functionality locally:
- added a cloud.provider.d vmware.conf template
- added a cloud.profile.d vmware.conf template
- altered cloud.sls to install pyvmomi if required

I'm raising this issue, because this functionality should probably exist in the cloud formula going forward.

if people are interested, i can prep a PR for review.

**Comment by nmadhok** [comment]
@cybacolt there's another formula for example of vmware cloud driver.

**Comment by cybacolt** [comment]
@nmadhok where is that? would it provide config and management of the provider through this salt cloud formula?

**Comment by nmadhok** [comment]
@cybacolt example still has to be uploaded for it. What do you mean by providing config and management of provider through this salt cloud formula?

**Comment by nmadhok** [comment]
@cybacolt my bad I didn't realize that cloud providers and the profile examples are also present in this formula. You're right that it should be added. Do you want to work on a PR? I can review it if you can submit one?

**Comment by cybacolt** [comment]
@nmadhok yeah, i'll have a go - tho looking at the full profile its quite complex, i'm not sure i can test all the edge cases.

also need to figure out how much of it needs to be configurable...
